docker/start.py

```diff
@@ -83,7 +83,7 @@ def generate_config_from_template(
                     handle.write(value)
             environ[secret] = value
 
-    environ["SYNAPSE_APPSERVICES"] = glob.glob("/data/appservices/*.yaml")
+    environ["SYNAPSE_APPSERVICES"] = glob.glob("/data/appservices/**/*.yaml", recurseive=True)
```

Comment:
```suggestion
    environ["SYNAPSE_APPSERVICES"] = glob.glob("/data/appservices/**/*.yaml", recursive=True)
```
Typo here. I don't think it would work at all with this typo, could you test that your change works in your Kubernetes environment please? It should be pretty straightforward to build&push an image with your patch to do so. Thanks !